apps/server/src/modules/common-cartridge/export/elements/v1.3.0/common-cartridge-organizations-wrapper-element.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { InternalServerErrorException } from '@nestjs/common';
+import {
+	createCommonCartridgeOrganizationElementPropsV130,
+	createCommonCartridgeOrganizationsWrapperElementPropsV130,
+} from '@shared/testing/factory/common-cartridge-element-props.factory';
+import { CommonCartridgeVersion } from '../../common-cartridge.enums';
+import { CommonCartridgeElementFactory } from '../common-cartridge-element-factory';
+import { CommonCartridgeOrganizationsWrapperElementV130 } from './common-cartridge-organizations-wrapper-element';
+
+describe('CommonCartridgeOrganizationsWrapperElementV130', () => {
+	describe('getSupportedVersion', () => {
+		describe('when using common cartridge version 1.3.0', () => {
+			const setup = () => {
+				const props = createCommonCartridgeOrganizationsWrapperElementPropsV130();
+				const sut = new CommonCartridgeOrganizationsWrapperElementV130(props);
+
+				return { sut };
+			};
+
+			it('should return correct version', () => {
+				const { sut } = setup();
+				const result = sut.getSupportedVersion();
+
+				expect(result).toBe(CommonCartridgeVersion.V_1_3_0);
+			});
+		});
+
+		describe('when using not supported common cartridge version', () => {
+			const notSupportedProps = createCommonCartridgeOrganizationsWrapperElementPropsV130();
+			notSupportedProps.version = CommonCartridgeVersion.V_1_1_0;
+
+			it('should throw error', () => {
+				expect(() => new CommonCartridgeOrganizationsWrapperElementV130(notSupportedProps)).toThrow(
+					InternalServerErrorException
+				);
+			});
+		});
+	});
+
+	describe('getManifestXmlObject', () => {
+		describe('when using common cartridge version 1.3.0', () => {
```

Comment:
They are 2 different outputs possible but only one is added in test. I expect that test coverage also show this.

Reply:
Here is only one output possible. Test coverage is 100%.